app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -0,0 +1,48 @@
+<%
+  id ||= "default-id-#{SecureRandom.hex(4)}"
+  heading_helper = GovukPublishingComponents::Presenters::HeadingHelper.new(local_assigns)
+
+  data_attributes ||= {}
+  data_attributes[:module] = 'accordion'
+
+  condensed ||= false
+  accordion_classes = 'gem-c-accordion govuk-accordion'
+  accordion_classes << ' govuk-accordion--condensed' if condensed
+
+%>
+<% if items.any? %>
+  <%= tag.div(class: accordion_classes, id: id, data: data_attributes) do %>
+    <% items.each_with_index do |item, i| %>
+      <%
+        index = i + 1
+
+        item[:data_attributes] ||= nil
+
+        section_classes = 'govuk-accordion__section'
+        section_classes << ' govuk-accordion__section--expanded' if item[:expanded]
+
+        summary_classes = 'govuk-accordion__section-summary'
+        summary_classes << (condensed ? ' govuk-body-s' : ' govuk-body')
+      %>
+      <div class="<%= section_classes %>">
+        <div class="govuk-accordion__section-header">
+          <%=
+            content_tag(
+              heading_helper.heading_tag,
+              content_tag('span', item[:heading][:text], class: "govuk-accordion__section-button", id: "#{id}-heading-#{index}", data: item[:data_attributes]),
+              class: 'govuk-accordion__section-heading'
+            )
+          %>
+          <% if item[:summary].present? %>
+            <div class="<%= summary_classes %>" id="<%= id %>-summary-<%= index %>">
```

Comment:
this can be put on one line by doing the following:

```
<% tag.div item[:summary][:text], class: summary_classes, id: "#{id}-summary-{index}" %>
```

Reply:
✅